src/modExcelToXML.bas

```diff
@@ -0,0 +1,169 @@
+Attribute VB_Name = "modExcelToXML"
+Option Explicit
+
+Const ms As Double = 0.000000011574
+
+Sub excelToXML()
```

Comment:
The error handling method used here seems like it will ignore all errors. I'm not sure if this is a good idea. What's your thoughts about this?